iptables/iptables.go

```diff
@@ -451,6 +457,17 @@ func (ipt *IPTables) runWithOutput(args []string, stdout io.Writer) error {
 	return nil
 }
 
+// lookIptablesDefaultPath searches the iptables binaries at the default paths.
+func lookIptablesDefaultPath(command string) (string, error) {
+	for _, prefix := range IPTABLES_DEFAULT_PREFIXES {
+		path := prefix + "/" + command
```

Comment:
Minor: this would better be a `filepath.Join()`.

Reply:
Done.